Firestore/src/CollectionReference.php

```diff
@@ -266,6 +267,17 @@ public function listDocuments(array $options = [])
             'mask' => []
         ];
 
+        if (isset($options['readTime'])) {
+            if (!($options['readTime'] instanceof Timestamp)) {
+                throw new \InvalidArgumentException(sprintf(
```

Comment:
Add `@throws...` phpdoc to every public function where this param is exposed. something like this:
```
* @throws \InvalidArgumentException if an invalid `$options.readTime` is specified.
```

Reply:
Thanks a lot for pointing that out, totally missed that.

Added in commit [41b2311](https://github.com/googleapis/google-cloud-php/pull/5865/commits/41b2311e14606643954616150399eaf1a13d0ba1).